content/applications/websites/website/publish/on-premise_geo-ip-installation.rst

```diff
@@ -5,60 +5,55 @@ Geo IP Installation (On-Premises Database)
 Installation
 ============
 
-.. warning::
-   Please note that the installation depends on your computer's operating system and distribution.
-   We will assume that a Linux operating system is being used.
+1. Download both the GeoLite2 City and Country
+   `databases <https://dev.maxmind.com/geoip/geoip2/geolite2/>`_. You should end up with two files
+   called ``GeoLite2-City.mmdb`` and ``GeoLite2-Country.mmdb``
```

Comment:
We have a task to highlight file names differently so let's start flagging them with the `file` role already :)
```suggestion
   called :file:`GeoLite2-City.mmdb` and file:`GeoLite2-Country.mmdb`
```